pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -190,21 +186,13 @@ func (m *kubeGenericRuntimeManager) generatePodSandboxLinuxConfig(pod *v1.Pod) (
 
 	if pod.Spec.SecurityContext != nil {
 		sc := pod.Spec.SecurityContext
-		if sc.RunAsUser != nil && runtime.GOOS != "windows" {
-			lc.SecurityContext.RunAsUser = &runtimeapi.Int64Value{Value: int64(*sc.RunAsUser)}
-		}
-		if sc.RunAsGroup != nil && runtime.GOOS != "windows" {
-			lc.SecurityContext.RunAsGroup = &runtimeapi.Int64Value{Value: int64(*sc.RunAsGroup)}
-		}
+		addLinuxSecurityContext(lc, sc)
 		namespaceOptions, err := runtimeutil.NamespacesForPod(pod, m.runtimeHelper)
```

Comment:
based on https://github.com/kubernetes/enhancements/pull/3275#issuecomment-1121603308 this also should be Linux only. No need to keep it here.

cc @rata

Reply:
Done

Reply:
LGTM